README.md

```diff
@@ -117,6 +117,7 @@ The following suites are available to run:
 * deploy
 * content
 * contrib
+* contrib-dev
```

Comment:
I need to expand on the definition of the suites. I agree, this needs more documentation.